contribs/drt/src/main/java/org/matsim/contrib/drt/run/DrtModeQSimModule.java

```diff
@@ -62,14 +62,21 @@ public DrtModeQSimModule(DrtConfigGroup drtCfg, AbstractQSimModule optimizerQSim
 
 	@Override
 	protected void configureQSim() {
-		boolean teleportDrtUsers = drtCfg.getDrtSpeedUpParams().isPresent() && DrtSpeedUp.isTeleportDrtUsers(
+		boolean teleportSpeedup = drtCfg.getDrtSpeedUpParams().isPresent() && DrtSpeedUp.isTeleportDrtUsers(
 				drtCfg.getDrtSpeedUpParams().get(), getConfig().controller(), getIterationNumber());
-		if (teleportDrtUsers) {
+
+		boolean teleportEstimate = drtCfg.getDrtEstimatorParams().isPresent() && drtCfg.simulationType == DrtConfigGroup.SimulationType.estimateAndTeleport;
+
+		if (teleportSpeedup) {
```

Comment:
Maybe the new approach could replace the old drt speedup (at least at some point)? Do you have any plans wrt the DRT speedup?

Reply:
You are right, we also think the old drt speedup might be replaced altogether. But we are not heavy users of that functionality and don't want to break it for now.  The only conceptual difference is that estimators are updated in speed-up, by performing a "full simulation". It should not be very difficult to also add this to our approach, but right now that was not our priority, and we have not tested that.